### DISCUSSION TITLE
Subsync Integration

### DISCUSSION THREAD

**Discussion by akang6** [pymedusa/Medusa#7692] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
You either have to only download exact match subtitles which often results in no subtitles at all or turn off exact match and get out of sync subtitles

**Describe the solution you'd like**
Subsync magically syncs out of sync subtitles. Having it integrated into Medusa so that it automatically runs it after downloading a subtitle would solve all my problems.

**Additional context**
https://github.com/smacke/subsync

**Comment by dirkronsmans** [comment]
+1 on this, seems awesome!

**Comment by p0psicles** [comment]
It has quite some dependencies itself. Maybe it would be possible to have users install it. And allow an easy hook if available?

**Comment by akang6** [comment]
My thought is that since I'm using the docker image I'd have to manually go and reinstall it every time there is an image update unless it's baked in to the image.

**Comment by p0psicles** [comment]
That would get backed in the image of course
